plugins/database/hana/hana_test.go

```diff
@@ -0,0 +1,172 @@
+package hana
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/builtin/logical/database/dbplugin"
+	"github.com/hashicorp/vault/plugins/helper/database/connutil"
+)
+
+var (
+	testHANAImagePull sync.Once
+)
+
+func TestHANA_Initialize(t *testing.T) {
```

Comment:
Just....to function?  At all?  8-O

There is at least one unofficial image, with the express edition

I wonder though if, given the lack of easy testing with anything official, we should actually pull this into an external repo like the oracle plugin. That way we can have a separate release cycle with more rapid releases if there are issues.

What do you two think?